systers_portal/common/tests/selenium/base.py

```diff
@@ -9,8 +9,9 @@
 from django.contrib.auth import (SESSION_KEY, BACKEND_SESSION_KEY,
                                  HASH_SESSION_KEY)
 from django.contrib.sessions.backends.db import SessionStore
+from allauth.account.models import EmailAddress
```

Comment:
Are the imports in any particular order ?

Reply:
I don't see any need for the imports to order. I don't think they are.

Reply:
It is a matter of personal choice, since PEP8 standards do not ask for it. But I believe for consistency it should be done. Here we have few imports, but when the code grows, and the imports increase in number, a sorted order comes in handy for the developers.